Dockerfile

```diff
@@ -1,67 +1,17 @@
-FROM registry.access.redhat.com/ubi8/nodejs-20:latest AS BUILD_IMAGE
-
-### BEGIN REMOTE SOURCE
-# Use the COPY instruction only inside the REMOTE SOURCE block
-# Use the COPY instruction only to copy files to the container path $REMOTE_SOURCES_DIR/activemq-artemis-self-provisioning-plugin/app
-ARG REMOTE_SOURCES_DIR=/tmp/remote_source
-RUN mkdir -p $REMOTE_SOURCES_DIR/activemq-artemis-self-provisioning-plugin/app
-WORKDIR $REMOTE_SOURCES_DIR/activemq-artemis-self-provisioning-plugin/app
-# Copy package.json and yarn.lock to the container
-COPY package.json package.json
-COPY yarn.lock yarn.lock
-ADD . $REMOTE_SOURCES_DIR/activemq-artemis-self-provisioning-plugin/app
-RUN command -v yarn || npm i -g yarn
-### END REMOTE SOURCE
```

Comment:
@lavocatt , We need all these changes added within the BEGIN to END remote source. because i added those lines to keep sync with the midstream and downstream repos. For now, could you please revert these changes?

Reply:
Can you point me to these repos please?
What is the purpose of these lines exactly?

Reply:
midstream self-provisioning-plugin repo: https://github.com/rh-messaging/activemq-artemis-self-provisioning-plugin

Dpwnstream self-provisioning-plugin repo: 
https://pkgs.devel.redhat.com/git/containers/amq-broker-self-provisioning-plugin

When I was working on automation for the self-provisioning-plugin during the downstream build to fix issues with the Dockerfiles, I synced all upstream->midstream->downstream Dockerfiles.

Reply:
All good now, we've discussed offline. The reason why this block is needed is because we only keep the Dockerfile in downstream and code gets injected at build time by another tool. So the upstream Dockerfile has to be compatible with that workflow. I'll add these lines back.